.changeset/eighty-papayas-exist.md

```diff
@@ -0,0 +1,5 @@
+---
+'@xstate/graph': patch
+---
+
+The `@xstate/graph` package now includes everything from `@xstate/test`.
```

Comment:
What are the current roles of both packages? I guess that with the "combine" in the title of this PR I expected this one would consolidate those 2 in a single package but we still have 2

Reply:
Deleted `@xstate/test` in #4911